apps/explorer/cypress/e2e/static/e2e.cy.ts

```diff
@@ -42,8 +42,15 @@ describe('End-to-end Tests', () => {
 
         it('includes the sender time information', () => {
             cy.visit(`/transactions/${successID}`);
+            // TODO - use the custom command date format function
             cy.get('[data-testid=transaction-timestamp]').contains(
-                'Dec 15, 2024, 00:00:00 UTC'
+                new Intl.DateTimeFormat('en-US', {
```

Comment:
Yeah this test isn't really great, it's really almost a unit test for the date formatting. We should probably just validate that it's present, not the contents. Although we need to really just move this to localnet tests.

Reply:
This is fine for this PR though, nothing immediately actionable.